### DISCUSSION TITLE
At mentions for todo list

### DISCUSSION THREAD

**Discussion by RickCogley** [outline/outline#1773] [Ideas]
I don’t think Outline has this, so I’m sorry if it does: it would be great if you could @ mention ppl in meeting notes action items, and have a page that lists all such mentions so it’s easy to follow up.

**Comment by tommoor** [comment]
Hey @RickCogley – we don't have mentions yet, it opens a bit of a can of worms in terms of notifications and their management plus like you say a way to list the TODO's that are created. Not to say it won't happen, I agree it would be nice to have

**Reply by RickCogley** [reply]
I wonder, instead of trying to make something general work anywhere, with logic like "if the @ mention is on a line that is a checkbox type line, then use regex to extract that out and copy it to a list over here", that, developing a bespoke "action item" slash command would not be better. That way it's structured, and you can safely have assumptions about it. (Although, it's also then opening another can of worms as you say, about due dates and so on)

     /action @tommoor do this-n-that

**Comment by almereyda** [comment]
Is it imaginable that @-mentions can be tied to Comments #457 for a starter?

A social annotation layer can be a useful workaround prior "socialising" the wiki text itself.
